Frameworks/UIKit/UIApplication.mm

```diff
@@ -168,38 +151,18 @@ - (UIView*)hitTest:(CGPoint)point withEvent:(UIEvent*)event {
 const NSTimeInterval UIApplicationBackgroundFetchIntervalMinimum = StubConstant();
 const NSTimeInterval UIApplicationBackgroundFetchIntervalNever = StubConstant();
 
-float statusBarHeight = 20.0f;
-static UIInterfaceOrientation _curOrientation = UIInterfaceOrientationPortrait;
-static UIInterfaceOrientation _internalOrientation = UIInterfaceOrientationPortrait;
-extern int requestDeviceOrientation;
 extern UIWindow* _curKeyWindow;
-
 NSArray<UIWindow*>* g_windows;
-UIWindow* popupWindow;
-UIApplication* sharedApplication;
-int showKeyboard, forceHideKeyboard;
-static int curKeyboardType, showKeyboardType;
-bool keyboardVisible;
-CGRect keyboardRect;
-UIView* _blankView;
-static bool blankViewUp = false;
-static float curBlankViewHeight = 0.0f;
-static bool doEvaluateKeyboard = false;
 
-// Right now it's expected that the user sets this when they want otherwise:
-float keyboardBaseHeight = 200, keyboardPhysicalHeight = 0;
+// TODO: #2443 Remove _popupWindow (and all references to it) when we move UIAlertView over to Xaml
+UIWindow* g_popupWindow;
```

Comment:
Yes :'( We have work tracking its removal though.

---
In reply to: [110294369](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110294369) [](ancestors = 110294369)